app/models/classification.rb

```diff
@@ -196,7 +196,7 @@ def self.get_tags_from_object(obj)
   end
 
   def self.create_category!(options)
-    self.create!(options.merge(:parent_id => 0))
+    is_category.create!(options)
```

Comment:
NOTE: this will not override a bad `:parent_id` passed in

Reply:
Doesn't `parent_id` need to be explicitly set to 0 in order for the `scope :is_category, -> { where(:parent_id => 0) }` to work when reading it? Or does `parent_id` get defaulted to 0 somewhere?

Reply:
@gtanzillo the `is_category` will default it to `0`, but also, the database has a default of `0` - so it is getting defaulted in 2 places.

I also added the [[test]](https://github.com/ManageIQ/manageiq/pull/18361/files#diff-5377fe1d33c36f7bdd9b5a7cf5937383R566) to ensure category (read: `parent_id`) is set correctly.